diff --git a/readthedocs/doc_builder/python_environments.py b/readthedocs/doc_builder/python_environments.py
index a8353d5f4..5042cbda0 100644
--- a/readthedocs/doc_builder/python_environments.py
+++ b/readthedocs/doc_builder/python_environments.py
@@ -270,13 +270,6 @@ class Virtualenv(PythonEnvironment):
             self.project.pip_cache_path,
         ]
 
-        # Install latest pip first,
-        # so it is used when installing the other requirements.
-        cmd = pip_install_cmd + ['pip']
-        self.build_env.run(
-            *cmd, bin_path=self.venv_bin(), cwd=self.checkout_path
-        )
-
         requirements = [
             'Pygments==2.2.0',
             # Assume semver for setuptools version, support up to next backwards
