diff --git a/readthedocs/search/tasks.py b/readthedocs/search/tasks.py
index cd82f8fe2..3d7efb4e7 100644
--- a/readthedocs/search/tasks.py
+++ b/readthedocs/search/tasks.py
@@ -23,7 +23,9 @@ def index_objects_to_es(
         raise ValueError('You can not pass both chunk and objects_id.')
 
     if not (chunk or objects_id):
-        raise ValueError('You must pass a chunk or objects_id.')
+        #raise ValueError('You must pass a chunk or objects_id.')
+        print('chunk: ', chunk)
+        print('objects_id: ', objects_id)
 
     model = apps.get_model(app_label, model_name)
     document = _get_document(model=model, document_class=document_class)
